README.md

```diff
@@ -256,8 +256,10 @@ To get started on building solutions on Aether, an [aether-bootstrap](https://gi
 
 ## Deployment
 
-Set the `HOSTNAME` and `CAS_SERVER_URL` environment variables if you want to
-activate the UMS integration in each container.
+The environment variable `HOSTNAME` is required.
+
+Set the environment variable `CAS_SERVER_URL` if you want to activate the UMS integration 
+in all django projects. Note that the UMS integration [also requires `HOSTNAME` to be set](https://github.com/eHealthAfrica/ums/blob/0561eeea74ebc1d937d58e5f212014bfeed4d6e8/documentation/django-ums-client.md#quick-start):
```

Comment:
I would not include any link to a private repo in the README, being this a public repo :grimacing:

Reply:
Yeah, forgot about that, removing now.